### PR TITLE
Removed leftover from tests

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Removed leftover from tests

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

### CODE REVIEW

File: tests/unit/app/endpoints/test_ols.py [hunk 1]
```diff
@@ -795,10 +795,6 @@ def test_generate_response_valid_subject(mock_summarize):
 def test_generate_response_on_summarizer_error(mock_summarize):
     """Test how generate_response function checks validation results."""
     # mock the DocsSummarizer
-    mock_response = Mock()
-    mock_response.response = (
-        "Kubernetes is an open-source container-orchestration system..."  # summary
-    )
     mock_summarize.side_effect = Exception  # any exception might occur
```
Comment (JoaoFula) [inline tests/unit/app/endpoints/test_ols.py:798]: Do we keep this line if we're removing the mock?

Comment (tisnik) [reply tests/unit/app/endpoints/test_ols.py:798]: yes, we need to raise an exception

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-service"
/override "Red Hat Konflux / lightspeed-service-on-pull-request"

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-service"

**Pull request merged**